src/release_metadata.rs

```diff
@@ -135,9 +137,19 @@ impl ReleaseMetadata {
             None
         };
 
-        let readme_path = flake_root.join(subdir).join("README.md");
-        let readme = if readme_path.exists() {
-            Some(tokio::fs::read_to_string(readme_path).await?)
+        let readme_dir = flake_root.join(subdir);
+
+        let readme = if let Some(Ok(path)) =
+            tokio::fs::read_dir(readme_dir)
+                .await?
+                .find(|entry| match entry {
+                    Ok(entry) => {
+                        entry.file_name().to_string_lossy().to_ascii_lowercase()
```

Comment:
I'm hesitant to do the _lossy part here, in case a weird path gets converted into the readme.md. I'd rather we do somethnig like make an OsStr from readme.md and compare that directly. Maybe like this: https://doc.rust-lang.org/stable/std/ffi/struct.OsStr.html#method.eq_ignore_ascii_case

Otherwise looks good, though as a trivial nit I think this bit here could deserve to be its own function.

Reply:
I was worried about this too, but it ought to be fine, since 

>[Any non-Unicode sequences are replaced with U+FFFD REPLACEMENT CHARACTER.](https://doc.rust-lang.org/stable/std/ffi/struct.OsStr.html#method.to_string_lossy)

and "readme.md" does not contain any of these. Maybe we should put that justification in a comment though.

Reply:
```suggestion
                        // to_string_lossy() replaces unknown sequences with U+FFFD REPLACEMENT CHARACTER, which doesn't occur in "readme.md", so this shouldn't result in any false positives.
                        entry.file_name().to_string_lossy().to_ascii_lowercase()
```